docs/client/NetworksApi.md

```diff
@@ -1,55 +1,16 @@
 # pureport\client\NetworksApi
 
-All URIs are relative to *http://api.pureport.com*
+All URIs are relative to *https://api.pureport.com*
 
 Method | HTTP request | Description
 ------------- | ------------- | -------------
-[**AddConnection**](NetworksApi.md#AddConnection) | **Post** /networks/{networkId}/connections | Add new connection
 [**AddNetwork**](NetworksApi.md#AddNetwork) | **Post** /accounts/{accountId}/networks | Add new network
 [**DeleteNetwork**](NetworksApi.md#DeleteNetwork) | **Delete** /networks/{networkId} | Delete network
 [**FindNetworks**](NetworksApi.md#FindNetworks) | **Get** /accounts/{accountId}/networks | List networks
-[**GetConnections**](NetworksApi.md#GetConnections) | **Get** /networks/{networkId}/connections | List network connections
 [**GetNetwork**](NetworksApi.md#GetNetwork) | **Get** /networks/{networkId} | Get network details
 [**UpdateNetwork**](NetworksApi.md#UpdateNetwork) | **Put** /networks/{networkId} | Update network
 
 
-# **AddConnection**
```

Comment:
Why did AddConnection() and GetConnections() disappear from the Network resource?

Reply:
Not sure about all of the Swagger voodoo here, but it looks like the `tags` from the Swagger Schema are used to assign methods to the generated API Services.
Not sure how or why Kato started tagging the swagger API differently, but if you look in the generated output, it was including the same API endpoint in both services:

https://github.com/pureport/pureport-sdk-go/blob/develop/pureport/client/api_connections.go#L402
https://github.com/pureport/pureport-sdk-go/blob/develop/pureport/client/api_networks.go#375

So basically just duplicates...

Reply:
Interesting. Maybe something changed that stopped triggering that? Just wondered.